p5js/NeuralNetwork/NeuralNetwork_titanic/sketch.js

```diff
@@ -1,45 +1,38 @@
-let classifier;
-let data;
-let normalizedData;
+let nn;
 let submitButton;
+let nnOptions = {
+  dataUrl: 'data/titanic_train-experimental.csv',
+  inputs: ['age', 'fare', 'is_female'],
+  outputs: ['survived_string'],
+  task: 'classification',
+  epochs: 50,
+  batchSize: 16
```

Comment:
I would also maybe leave `batchSize` out of this example and just let it use a default for simplicity. I might also suggest moving this to:

```javascript
classifier.data.setBatchSize(16);
```

Reply:
Yep this is also resolved.

we do:

```js
const trainingOptions ={
 batchSize: 50,
epochs: 100
}
nn.train(options, finishedTraining);
```